srtcore/core.cpp

```diff
@@ -8463,7 +8463,7 @@ void CUDT::processCtrl(const CPacket &ctrlpkt)
 
     case UMSG_CGWARNING: // 100 - Delay Warning
         // One way packet delay is increasing, so decrease the sending rate
-        m_tdSendInterval *= 1.125;
+        m_tdSendInterval = (m_tdSendInterval * 1125) / 1000;
```

Comment:
As you touched this field already, it would be nice to determine if this makes sense any longer. According to the current code it will be set again at the next ACK or CHECKTIMER events in `updateCC`. It's thought of as an emergency break, but it will only last a while and this slowdown will not be taken into account by CC, unless it is doing it separately on its own.

Reply:
I think we even don't send `UMSG_CGWARNING` as of now. So this code is just maintained.

Reply:
Would be nice to have some information added here then. The most sensible would be to connect the slot for `TEV_ACKACK` in the `FileCC` class, which is the most proper place to react on changing RTT. Might be also that the instantaneous value of RTT could be passed as argument here.